### PR TITLE
upload homework02

### PR DESCRIPTION
**Pull request by ghostbinsun**

### CODE REVIEW

File: python_basics/class03/exercise_6.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    sorted_str: string in a hyphen-separated sequence after 
+    sorting them alphabetically
+    '''
+
+    # code up your solution here
+    
+    list2=hyphen_str.split('-')
+    list2 = sorted(list2)
+
+    sorted_str=''
```
Comment (emma-oc) [inline python_basics/class03/exercise_6.py:74]: this works; using `.join` might be a more efficient way

File: python_basics/class03/exercise_6.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    Argument:
+    number: number to check
+
+    Returns:
+    perfect: boolean, True if number is perfect
+    '''
+
+    # code up your answer here
+    list=[]
+    for i in range (2, number):
```
Comment (emma-oc) [inline python_basics/class03/exercise_6.py:108]: this works; you can also start with 1 and the final sum won't need the `+1`

**Comment by emma-oc** [comment]
Good job, some minor changes

File: python_basics/class03/exercise_6.py (hunk 1 above)
Comment (ghostbinsun) [reply python_basics/class03/exercise_6.py:74]: got it thanks

**Pull request closed without merging**